src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/mapper/CaseDataOtherChildrenDetailsElementsMapper.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.prl.controllers.citizen.mapper;
+
+import uk.gov.hmcts.reform.prl.enums.DontKnow;
+import uk.gov.hmcts.reform.prl.enums.Gender;
+import uk.gov.hmcts.reform.prl.models.Element;
+import uk.gov.hmcts.reform.prl.models.c100rebuild.C100RebuildOtherChildrenDetailsElements;
+import uk.gov.hmcts.reform.prl.models.c100rebuild.ChildDetail;
+import uk.gov.hmcts.reform.prl.models.c100rebuild.DateofBirth;
+import uk.gov.hmcts.reform.prl.models.c100rebuild.PersonalDetails;
+import uk.gov.hmcts.reform.prl.models.complextypes.Child;
+import uk.gov.hmcts.reform.prl.models.dto.ccd.CaseData;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+import static uk.gov.hmcts.reform.prl.enums.YesOrNo.Yes;
+
+public class CaseDataOtherChildrenDetailsElementsMapper {
+
+    public static void updateOtherChildDetailsElementsForCaseData(CaseData.CaseDataBuilder caseDataBuilder,
+                                                                  C100RebuildOtherChildrenDetailsElements c100RebuildOtherChildrenDetailsElements) {
+
+        if ("Yes".equals(c100RebuildOtherChildrenDetailsElements.getHasOtherChildren())) {
```

Comment:
@saranyavinayagam8 Can we please use 'Yes' from PrlAppsConstants ?

Reply:
done